C/ColPack/build_tarballs.jl

```diff
@@ -0,0 +1,53 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "ColPack"
+version = v"0.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/michel2323/ColPack.git", "c8674588c8204e9f05a2c6adea21a28f7b56f490")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd ColPack/build/automake/
+autoreconf -vif
+mkdir build
+cd build/
+../configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+make -j${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
```

Comment:
This should be simply
```julia
platforms = supported_platforms(; exclude=p -> Sys.isappl(p) || Sys.iswindows(p))
```
What's wrong with those platforms?

Reply:
OpenMP. I think it might be related to what you outlined above. Thanks for highlighting that. I'll look into it.

Reply:
I tried to fix OpenMP for Apple but I can't figure it out. It cannot find 'omp.h'. Forcing `gcc` lead to linking errors. I see that other projects have also issues with Apple and OpenMP. On Windows it fails with some linking errors.